app/lib/common/providers/notifiers/chat_notifiers.dart

```diff
@@ -86,54 +86,66 @@ class ChatRoomsListNotifier extends StateNotifier<List<Convo>> {
 
   List<Convo> listCopy() => List.from(state, growable: true);
 
+  void _sortList(List<Convo> list) {
+    list.sort((a, b) => b.latestMessageTs().compareTo(b.latestMessageTs()));
+  }
+
```

Comment:
May I suggest to not do this here but instead put a provider in between this notifier and the actual list to be displayed and make that resort the list instead? This way we always have the proper indexed list (as the rust-sdk requires) and we can always resort on the UI for the proper displaying.

Reply:
Sounds good.